### PR TITLE
update to work with windows

### PR DESCRIPTION
**Pull request by caseyyee**

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function log() {
 }
 
 function fixSettings( settings ) {
-
+  var defaultSettings = {};
```
Comment (caseyyee) [inline common.js:14]: @cvan not sure where this data comes from.   undefined otherwise.   Ideas?

Comment (cvan) [reply common.js:14]: I thought I removed this, yeah remove. should add `jshint` at some point.

**Comment by caseyyee** [comment]
@cvan r?   Looks like window load already happens by the time polyfill.js loads, so we're check document readystate instead.

**Comment by cvan** [comment]
thanks!

**Pull request merged**